dali/operators/image/crop/crop_mirror_normalize.h

```diff
@@ -133,6 +131,11 @@ class CropMirrorNormalize : public Operator<Backend> {
         inv_std_vec_.resize(args_size, inv_std_vec_[0]);
     }
 
+    for (int i = 0, d = args_size; i < d; i++) {
+      mean_vec_[i] -= shift * inv_std_vec_[i] / scale;
```

Comment:
Does it make any sense to calculate inv_scale and change the division to multiplication?

Reply:
It's just the constructor. There are just a few values to process (typically 3) - precision is much more important than performance here. I even considered running these calculations in double precision.

Reply:
Ok.